### ISSUE TITLE
Create usage analyzer

### ISSUE THREAD

**Issue by lobobabysaurus** [lobobabysaurus/NewAppearances#19]
Create an app module that reads in a bunch of information about visiting users  whenever the site is hit and adds it to the database in a storage effective way.  

Needs to be able to filter out return users.

Once the database gathering aspect is done we need to have an effective display for it.

I was thinking drillable pie charts.

----------------------------------------
- Bitbucket: https://bitbucket.org/philsim7/newappearances/issue/6
- Originally reported by: Phil Simmons
- Originally created at: 2015-02-07T17:01:51.679

**Comment by lobobabysaurus** [comment]
On hold until we get closer to Production, but should definitely check out D3.js and start looking into other libraries

----------------------------------------
Original comment by: Phil Simmons
